.github/actions/benchexec-test/action.yml

```diff
@@ -34,15 +34,14 @@ runs:
         unzip theta.zip
         mv Theta theta
     - uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # v3.0.2
-      name: Get JAR
+      name: Get zip
       with:
-        name: ThetaJars
-        path: jar/
-    - name: Add new jar to archive
+        name: Theta_SV-COMP
+        path: release/
+    - name: Unzip 
```

Comment:
Are these changes intentional? Similarly, are the changes in this file are intentional?